lib/checkCriticalPullRequest.js

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to handle checks when a PR adds/modifies the data storage layer.
+ */
+
+const {
+  CRITICAL_LABEL,
+  getAllChangedFiles,
+  getNameString,
+  getNewItemsFromFileByRegex,
+  hasCriticalLabel
+} = require('./utils');
+
+const { teamLeads, oppiaMaintainers } = require('../userWhitelist.json');
+const STORAGE_DIR_PREFIX = 'core/storage/';
+const MODEL_REGEX = new RegExp(
+  [
+    '(?<addition>\\+)(?<classDefinition>class\\s)',
+    '(?<name>[a-zA-Z]{2,256})(?<suffix>Model)(?<funDef>\\()',
+  ].join('')
+);
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const getNewModelsFromFile = (file) => {
+  return getNewItemsFromFileByRegex(MODEL_REGEX, file);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const addsNewModel = (file) => {
```

Comment:
All function names should start with third-person plural -- "add" not "adds". Please check the others too. Also make sure to include docstrings for each function to explain what it is doing, similar to Python.